Cargo.toml

```diff
@@ -12,27 +12,30 @@ rust-version = "1.75.0"
 [workspace.dependencies]
 bincode = "1.3"
 criterion = "0.5"
-indexmap = "2.2"
+indexmap = "2.6"
 itertools = "0.12"
 litesvm = { path = "svm", version = "0.2" }
 log = "0.4"
 serde = "1.0"
 smallvec = "1.13"
-solana-address-lookup-table-program = "~1.18"
-solana-bpf-loader-program = "~1.18"
-solana-compute-budget-program = "~1.18"
-solana-config-program = "~1.18"
-solana-loader-v4-program = "~1.18"
-solana-program = "~1.18"
-solana-program-runtime = "~1.18"
-solana-program-test = "~1.18"
-solana-sdk = "~1.18"
-solana-stake-program = "~1.18"
-solana-system-program = "~1.18"
-solana-vote-program = "~1.18"
-spl-associated-token-account = "3.0"
-spl-token = "3.5"
-spl-token-2022 = "3.0"
+solana-address-lookup-table-program = "2.0.13"
```

Comment:
Why will we don't use 2.0  instead of 2.0.13 here since solana is following semver right now ?